content/academy/web_scraping_for_beginners.md

```diff
@@ -1,34 +1,65 @@
 ---
-title: Web scraping for beginners
-description: Learn what web scraping and crawling is and how to develop your own simple scraper.
-menuWeight: 19
-category: web scraping academy
+title: Web Scraping for Beginners
```

Comment:
Sure, I can change it. But I thought that if it's the name of the course, it should be using caps. That's not the case?

Reply:
Yeah, in academia they capitalize the titles but I'd say for our purposes it'll be obvious it's the course title since it's in the `courses` category, so let's go for consistency here